examples/dbscan/ArborX_DBSCAN.hpp

```diff
@@ -195,130 +193,49 @@ void dbscan(ExecutionSpace exec_space, Primitives const &primitives,
   // The finalization kernel will, ultimately, make all parents
   // point directly to the representative.
   // ```
-  Kokkos::parallel_for("ArborX::DBSCAN::flatten_stat",
+  Kokkos::View<int *, MemorySpace> cluster_sizes(
+      "ArborX::dbscan::cluster_sizes", n);
+  Kokkos::parallel_for("ArborX::dbscan::finalize_labels",
                        Kokkos::RangePolicy<ExecutionSpace>(exec_space, 0, n),
                        KOKKOS_LAMBDA(int const i) {
                          // ##### ECL license (see LICENSE.ECL) #####
                          int next;
-                         int vstat = stat(i);
+                         int vstat = labels(i);
```

Comment:
you can also alias the view.  I seem to remember you insisted on the name when you originally wrote this.

Reply:
Yeah, I could. But I decided to change, as I always had trouble remembering what `stat` was. And now with `labels` around, it would have become even more confusing. Does not really change the ECL part of the code, I'm ok with this kind of change here.